src/components/HeaderBar/Logo.js

```diff
@@ -4,6 +4,8 @@ import PureRender from 'pure-render-decorator';
 
 import styles from './Logo.less';
 
+const img = 'https://raw.githubusercontent.com/drets/heutagogy-chrome-extension/master/chrome/assets/img/icon-128.png';
```

Comment:
Link to the official repo and not @drets' fork. (Or better copy resource to this repo.)